modular_skyrat/modules/teshari/modules/mob/living/carbon/human/species_types/teshari.dm

```diff
@@ -1,9 +1,9 @@
 #define TESHARI_TEMP_OFFSET -30 // K, added to comfort/damage limit etc
-#define TESHARI_BURNMOD 1.5 // They take more damage from practically everything
-#define TESHARI_BRUTEMOD 1.5
-#define TESHARI_HEATMOD 1.5
+#define TESHARI_BURNMOD 1.25 // They take more damage from practically everything
+#define TESHARI_BRUTEMOD 1.2
+#define TESHARI_HEATMOD 1.3
 #define TESHARI_COLDMOD 0.67 // Except cold.
-#define TESHARI_PUNCH_LOW 1 // Lower bound punch damage
+#define TESHARI_PUNCH_LOW 2 // Lower bound punch damage
```

Comment:
Undocumented buff

Reply:
wait yeah why don't you say that you buffed potential punch damage for them?